docsite/rst/galaxy.rst

```diff
@@ -1,105 +1,104 @@
 
 .. contents:: Topics
 
 The Website
 ```````````
 
-The website `Ansible Galaxy <https://galaxy.ansible.com>`_, is a free site for finding, downloading, and sharing community developed Ansible roles. Downloading roles from Galaxy is a great way to jumpstart your automation projects.
+`Galaxy <https://galaxy.ansible.com>`_, is a free site for finding, downloading, and sharing community developed roles. Downloading roles from Galaxy is 
+a great way to jumpstart your automation projects.
 
-Access the Galaxy web site using GitHub OAuth, and to install roles use the 'ansible-galaxy' command line tool included in Ansible 1.4.2 and later.
+You can also use the site to share roles that you create. By authenticating with the site using your GitHub account, you're able to *import* roles, making 
+them available to the Ansible community. Imported roles become available in the Galaxy search index and visible on the site, allowing users to 
+discover and download them.
 
-Read the "About" page on the Galaxy site for more information.
+Learn more by viewing `the About page <https://galaxy.ansible.com/intro>`_.
 
-The ansible-galaxy command line tool
-````````````````````````````````````
+The command line tool
+`````````````````````
 
-The ansible-galaxy command has many different sub-commands for managing roles both locally and at `galaxy.ansible.com <https://galaxy.ansible.com>`_.
+The ``ansible-galaxy`` command comes bundled with Ansible, and you can use it to install roles from Galaxy or directly from a git based SCM. You can 
+also use it to create a new role, remove roles, or perform tasks on the Galaxy website. 
 
-.. note::
-
-    The search, login, import, delete, and setup commands in the Ansible 2.0 version of ansible-galaxy require access to the 
-    2.0 Beta release of the Galaxy web site available at `https://galaxy-qa.ansible.com <https://galaxy-qa.ansible.com>`_.
-
-    Use the ``--server`` option to access the beta site. For example::
-
-        $ ansible-galaxy search --server https://galaxy-qa.ansible.com mysql --author geerlingguy
-
-    Additionally, you can define a server in ansible.cfg::
-
-        [galaxy]
-        server=https://galaxy-qa.ansible.com
 
 Installing Roles
 ----------------
 
-The most obvious use of the ansible-galaxy command is downloading roles from `the Ansible Galaxy website <https://galaxy.ansible.com>`_::
+Use the ``ansible-galaxy`` command to download roles from the `Galaxy website <https://galaxy.ansible.com>`_
 
-   $ ansible-galaxy install username.rolename
+::
+
+    $ ansible-galaxy install username.role_name
 
 roles_path
 ==========
 
-You can specify a particular directory where you want the downloaded roles to be placed::
+Be aware that Ansible downloads roles to *ANSIBLE_ROLES_PATH*, which defaults to */etc/ansible/roles*. You can override this by setting the environment 
+variable in your session, defining *roles_path* in an *ansible.cfg* file, or by using the *--roles-path* option.
```

Comment:
roles path can be a series of directories i.e. `/etc/ansible/roles:~/.ansible/roles`, the default is correct but it should install to the 'first writable path configured'